README.md

```diff
@@ -10,6 +10,18 @@ Secrets are encrypted in this repository using [SOPS](https://github.com/mozilla
 
 If you'd like to change anything about hashbang's infrastructure, please send a PR!
 
+## Setup
+
+### Dependencies
+
+1. kubectl
+2. sops
+3. [ksops][ksops]
+4. gnupg
+
+For encrypting passwords please ensure that you import the Hashbang ArgoCD GPG Key.
+This can be done by running `gpg --import extra/deploy-key.pub`
```

Comment:
this should be extras (with an s).

However I like the current setup that everything is an app. Can we put it in the argocd folder? or the root?